src/main/java/oss/fosslight/controller/OssController.java

```diff
@@ -145,7 +145,24 @@ public String list(HttpServletRequest req, HttpServletResponse res, Model model)
 		
 		ossMaster.setCurPage(page);
 		ossMaster.setPageListSize(rows);
-		
+
+		String homepage =req.getParameter("homepage");
+		String[] urls = new String[2];
+		// download location 변환
+		if(!homepage.startsWith("https://")){
+			if(homepage.equals("http://")|| homepage.equals("www") || homepage.equals("")){
+				ossMaster.setHomepage("https://");
+			}else{
+				if(homepage.startsWith("http://")){
+					urls= homepage.split("://");
+				}
+				if(homepage.startsWith("www.")){
+					urls= homepage.split("www.");
+				}
+				ossMaster.setHomepage("https://"+urls[1]);

```

Comment:
If `http://` or `www` is not in included, homepage variable is `https://null`. 

<img width="797" alt="스크린샷 2021-08-31 오후 6 09 13" src="https://user-images.githubusercontent.com/17478634/131478740-82c342f0-09cd-4fe6-8923-353464c21d9e.png">

Reply:
Dear @epicarts ,
first of all thank you for let me know about this.
i fixed the case that homepage url doesn't start with http or www .

- test results
<img width="1474" alt="스크린샷 2021-08-31 오후 10 40 45" src="https://user-images.githubusercontent.com/46369622/131513335-f6434b85-e5e8-4714-86e3-817e88319a52.png">
<img width="1671" alt="스크린샷 2021-08-31 오후 10 28 35" src="https://user-images.githubusercontent.com/46369622/131513346-0a025ae3-fd7c-49e8-9796-de9c6e6a69fc.png">